context.go

```diff
@@ -921,6 +949,8 @@ func (c *Context) Pow(d, x, y *Decimal) (Condition, error) {
 
 	// decNumber sets the precision to be max(x digits + exponent, c.Precision)
 	// + 4. 6 is used as the exponent digits.
+	// TODO(mjibson): what is decNumber, and what is the source of the +4?
```

Comment:
decNumber is the reference implementation from the GDA site. decNumber is referred to in a few parts and I don't think it needs a specific explanation here. I'm not sure where they got the + 4. The comment about the 6 is the same as in the decNumber code, and I'm not able to explain that either. \o/ But the tests pass so...?